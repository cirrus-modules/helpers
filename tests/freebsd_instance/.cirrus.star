load("../../lib.star", "task", "freebsd_instance")

def main(ctx):
    return [
        task("freebsd task", freebsd_instance("freebsd-13-0")),
        task("freebsd task with image_name",
             freebsd_instance(image_name="family/freebsd-13-0")),
    ]
