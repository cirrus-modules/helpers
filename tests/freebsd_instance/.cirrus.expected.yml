task:
  name: freebsd task
  freebsd_instance:
    image_family: freebsd-13-0
task:
  name: freebsd task with image_name
  freebsd_instance:
    image_name: family/freebsd-13-0
